go.mod

```diff
@@ -17,13 +17,13 @@ require (
 	go.uber.org/zap v1.24.0
 	golang.org/x/sys v0.8.0
 	gomodules.xyz/jsonpatch/v2 v2.2.0
-	k8s.io/api v0.27.1
-	k8s.io/apiextensions-apiserver v0.27.1
-	k8s.io/apimachinery v0.27.1
-	k8s.io/client-go v0.27.1
-	k8s.io/component-base v0.27.1
-	k8s.io/klog/v2 v2.90.1
-	k8s.io/utils v0.0.0-20230209194617-a36077c30491
+	k8s.io/api v0.27.2
+	k8s.io/apiextensions-apiserver v0.27.2
+	k8s.io/apimachinery v0.27.2
+	k8s.io/client-go v0.27.2
+	k8s.io/component-base v0.27.2
+	k8s.io/klog/v2 v2.100.1
```

Comment:
Can we keep this on the same version used by Kubernetes? https://github.com/kubernetes/kubernetes/blob/v1.27.2/go.mod#L112

Reply:
reverted